core/trino-main/src/test/java/io/trino/memory/TestLeastWastedEffortLowMemoryTaskKiller.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.memory;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.airlift.units.DataSize;
+import io.airlift.units.Duration;
+import io.trino.execution.TaskId;
+import io.trino.execution.TaskInfo;
+import io.trino.execution.TaskState;
+import io.trino.execution.TaskStatus;
+import io.trino.execution.buffer.BufferState;
+import io.trino.execution.buffer.OutputBufferInfo;
+import io.trino.operator.TaskStats;
+import org.joda.time.DateTime;
+import org.testng.annotations.Test;
+
+import java.net.URI;
+import java.util.Map;
+import java.util.Optional;
+
+import static io.trino.memory.LowMemoryKillerTestingUtils.taskId;
+import static io.trino.memory.LowMemoryKillerTestingUtils.toNodeMemoryInfoList;
+import static io.trino.memory.LowMemoryKillerTestingUtils.toRunningQueryInfoList;
+import static io.trino.testing.assertions.Assert.assertEquals;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class TestLeastWastedEffortLowMemoryTaskKiller
```

Comment:
nit: `TestLeastWastedEffortTaskLowMemoryKiller` (same for other tests)

Reply:
thx; others were fine